gateway/1.4/modules/ROOT/pages/_partials/task-reg-run-flex-gateway.adoc

```diff
@@ -77,9 +77,27 @@ to generate set of instructions that includes a command block with the registrat
 // tag::reg-command-intro[]
 To register a Flex Gateway with Anypoint Platform, you must enter the registration command and then the start command. Each command includes information specific to your Anypoint Platform instance and *must be updated before* executing. See <<Before You Begin>> for more information on how to find the information you will need.
 //end::reg-command-intro[]
+//
+// tag::k8s-connected-intro[]
+In this mode, the gateway is managed entirely through Anypoint Platform, where you can: 
```

Comment:
I suggest using a softer term than `entirely` because there are some configurations that must be provided as local files/k8s resources.
 
```suggestion
In this mode, the gateway is managed mainly through Anypoint Platform, where you can: 
```

Reply:
@seblaz If we use "mainly" here, we'll need to very specific about what is managed where. Words like "mainly" makes the readers ask questions.